model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -2219,7 +2216,7 @@ def infer_hardware_from_model_name(model_name: str) -> CreateDockerImageBatchJob
             )
 
         b_params = int(numbers[-1])
-        if b_params <= 7:
+        if b_params <= 8:
```

Comment:
This magic number seems a bit brittle.

Reply:
best scenario i need to come up with a formula that incorporates # billions of parameters and context window length (for MoE add # of experts, # billion of parameters shared and for each expert), and output GB memory needed, from there I can calculate GPU type and sharding scheme. i feel this is too much work to justify

Reply:
will do a bit more calculation and less hardcode